mongodb/devhub
# Update and Fix Storybook Configuration

This PR builds off of Jensen's work in #259 and finishes adding in fonts and other styles to have components render as expected and finish setting up storybook for devhub. There should not be any changes to the actual devhub site.

To test:
- Run `yarn storybook` (yep yarn works)
- Check out the storybook!

The changes I made after #259 to get this going were:
- Took style declarations in `layout.js` and applied them to `.storybook/preview-head.html` so they would be picked up by each component. `preview-head.html` is evaluated before each component is rendered, so our "global" styles will be picked up by each story component.
- Added `import 'typeface-fira-mono'` to the storybook `theme.js` file so our proper typeface is picked up (this also happens in `layout.js`), a different Fira Mono was being used for some reason.
- Cleaned up some broken stories (charts id was out of date, series was out of date)
